packages/app/cypress/e2e/subscriptions/errorWarningChange-subscription.cy.ts

```diff
@@ -62,7 +62,7 @@ module.exports = {
 
         assertLoadingIntoErrorWorks('SyntaxError')
         cy.contains('Your configFile is invalid').should('be.visible')
-        cy.contains('Unexpected token \',\'').should('be.visible')
+        cy.contains('Unexpected token \',\'').scrollIntoView().should('be.visible')
```

Comment:
This is now necessary, because we are (I would argue correctly) showing the stack trace in this view which pushes 'Unexpected token' off screen.